parallel/tutorial.md

```diff
@@ -38,111 +38,121 @@ threads can be used simultaneously to do the computation, which comprises the se
 parallelization strategy.
 
 # Running the code
```

Comment:
How about putting the "Test Scenarios" before this point? Most users will just want to know how to use the parallelization and won't care about running performance tests themselves.